doc/chapters/filesystem/index.rst

```diff
@@ -0,0 +1,102 @@
+Filesystems
+***********
+
+This chapter documents the behavior and layout of the basic filesystem of the
+containers and where it picks up the different packages from. 
+
+
+Standard filesystem
+===================
+
+The base filesystem is bind mounted from a directory created by lxc, the
+directory is empty by default and is setup by the ``lxc-softwarecontainer.in``
```

Comment:
When I read this I just wondered if this is the level this documentation should be on. Perhaps mentioning the exact script is more the level of detail we want in the Developer chapter? Or perhaps in both chapters.